frontend/view-edit-client/client-section.component.tsx

```diff
@@ -5,14 +5,24 @@ import AuditSummarySection from "./audit-summary-section.component";
 import { LastUpdate } from "./view-client.component";
 
 export default function ClientSection(props: ClientSectionProps) {
-  const [expanded, setExpanded] = React.useState(true);
+  const [expanded, setExpanded] = React.useState(() =>
+    localStorage.getItem("...") ? JSON.parse(localStorage.getItem("...")) : true
+  );
   const scope = useCss(css);
 
+  const toggleExpandAndStore = () => {
+    localStorage.setItem(
+      `cu-client-section-expanded:${props.title}`,
+      JSON.stringify(!expanded)
+    );
+    setExpanded(!expanded);
+  };
+
   return (
     <div className="card padding-0" {...scope}>
       <button
         className="unstyled client-section-header"
-        onClick={() => setExpanded(!expanded)}
+        onClick={() => toggleExpandAndStore}
```

Comment:
This is a bug. It should be `onClick={toggleExpandAndStore}`. Your code here won't work because the `toggleExpandAndStore` function is not being called. If you pass it in as the entire onClick function, React will call it for us. But if you pass in an arrow function as the onClick, you have to call it.

My suggestion earlier was just to avoid creating an arrow function when you don't need it.